packages/p2p-v2/src/components/FullPageMobileWrapper/FullPageMobileWrapper.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { PropsWithChildren } from 'react';
+import clsx from 'clsx';
+import LeftArrowIcon from '../../public/ic-arrow-left.svg';
+import './FullPageMobileWrapper.scss';
+
+type TFullPageMobileWrapperProps = {
+    renderFooter?: () => React.ReactNode;
+    renderHeader?: () => React.ReactNode;
+    shouldFixedFooter?: boolean;
+};
+
+const FullPageMobileWrapper = ({
+    children,
+    renderFooter,
+    renderHeader,
+    shouldFixedFooter = true,
+}: PropsWithChildren<TFullPageMobileWrapperProps>) => {
+    return (
+        <div
+            className={clsx('p2p-v2-mobile-wrapper', {
+                'p2p-v2-mobile-wrapper--fixed-footer': shouldFixedFooter,
+            })}
+        >
+            {renderHeader && (
+                <div className='p2p-v2-mobile-wrapper__header'>
+                    <LeftArrowIcon />
+                    {renderHeader?.()}
```

Comment:
since its already checking for renderHeader, and renderFooter, if its true, we don't need the ? right?
```suggestion
                    {renderHeader()}
```